src/main/java/core/basesyntax/dao/jdbc/DaoJdbcDriverImpl.java

```diff
@@ -55,8 +58,8 @@ public Driver create(Driver item) {
 
     @Override
     public List<Driver> getAll() {
-        String query = "SELECT id, name, license_number FROM drivers WHERE deleted = false "
-                + "ORDER BY id";
+        String query = "SELECT id, name, license_number, login, password FROM drivers "
```

Comment:
```suggestion
        String query = "SELECT id, name, license_number, login FROM drivers "
```
ideally, it's better not to return a users' password from back end